zombie-bite-scripts/authorize_upgrade_ah.ts

```diff
@@ -60,15 +60,15 @@ const message = XcmVersionedXcm.V4([
   XcmV4Instruction.Transact({
     origin_kind: XcmV2OriginKind.Superuser(),
     require_weight_at_most: {
-      ref_time: 999999999n,
-      proof_size: 7777777n,
+      ref_time: 800000000,
```

Comment:
```suggestion
      ref_time: 800000000,
```

Better to make these bigger than we think so the script won't fall over if they change. This hardcoded weight is removed in v5 (which is what will be running for the real upgrade).

AFAIU you can use V5 instructions already and provide a backup weight which will be executed on chains on V4 or lower

Reply:
I was getting: downwardQueue.overweight error or something like that thus lowered the numbers.

I'll refactor script to v5

Reply:
May I submit as is? @seadanda

Reply:
ah yea they'll need to be reasonable and within the limits, but I assumed you meant that you took the weights from the weights file, so just mean to add a healthy buffer to those. Fine to merge this now